src/components/AddLink.tsx

```diff
@@ -81,33 +109,51 @@ const AddLinkButton = styled.button`
   white-space: nowrap;
 `;
 
-const AddLink = ({ folder }) => {
-  const AddLink = useRef(null);
+const Observed = styled.div`
+  height: 1px;
+  margin-bottom: 53px;
+  &.big {
+    height: 137px;
+
+    @media (max-width: 767px) {
+      height: 105px;
+    }
+  }
+`;
+
+const AddLink = ({ folder, footView }: { folder: any; footView: boolean }) => {
```

Comment:
이런 방식으로 작성하면 가독성이 더 좋을 것 같습니다

```suggestion
interface AddLinkPrpos { 
 folder: any; 
 footView: boolean;
}

const AddLink = ({ folder, footView }: AddLinkProps) => {
```